src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/OxoGReadCounts.java

```diff
@@ -34,26 +38,16 @@
  */
 public final class OxoGReadCounts extends GenotypeAnnotation {
 
-    public static final Allele REF_C = Allele.create(C.base, true);
-    public static final Allele REF_A = Allele.create(A.base, true);
-
     @Override
     public List<String> getKeyNames() {
-        return Arrays.asList(GATKVCFConstants.OXOG_ALT_F1R2_KEY,
-                             GATKVCFConstants.OXOG_ALT_F2R1_KEY,
-                             GATKVCFConstants.OXOG_REF_F1R2_KEY,
```

Comment:
For this and the above comment, how about I just make this `VCFHeaderLineCount.R` instead of `VCFHeaderLineCount.A`?